protractor-typescript-cucumber/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "protractor-typescript-cucumber",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "To demostrate protractor cucumber tests with typescript",
+  "keywords": [
+    "protractor",
+    "cucumber",
+    "typescript",
+    "angular",
+    "angularjs",
+    "testing",
+    "behaviour driven development",
+    "bdd",
+    "selenium",
+    "webdriverJS",
+    "gherkin",
+    "automation testing"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "tsc": "./node_modules/typescript/bin/tsc",
+    "pretest": "webdriver-manager update && npm run tsc",
+    "test": "protractor tmp/Config/config.js"
+  },
+  "author": "Ram Pasala <ram.pasala7@gmail.com>",
```

Comment:
does this project still have to mention as author Ram ? considering its Angular repo and Apache License. Should we revisit ? Just double checking.